MTVd-test-project/index.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html class="no-js">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title> MITv </title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="../css/main.css">
```

Comment:
Same as in the previous project, keep files in a folder under the current level, and do not use dots.